kf_model_fhir/ingest_plugin/target_api_builders/sequencing_observation.py

```diff
@@ -0,0 +1,142 @@
+"""
+Builds FHIR Observation resources (https://www.hl7.org/fhir/observation.html)
+from rows of tabular sequencing experiment data.
+"""
+import json
+
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+
+from kf_model_fhir.ingest_plugin.target_api_builders.kfdrc_genomic_file import (
+    GenomicFile,
+)
+from kf_model_fhir.ingest_plugin.shared import join
+
+class SequencingObservation:
+    class_name = "sequencing_observation"
+    resource_type = "Observation"
+    target_id_concept = None
+
+    @staticmethod
+    def build_key(record):
+        assert None is not record[CONCEPT.SEQUENCING.ID]
+        assert False is not any(
```

Comment:
Are these all of the required fields? Wouldn't this make sense to put in `build_entity` since it doesn't have anything to do with building the UK?

Reply:
As discussed, I will change this to:

```python
        assert True is any(
            [
                record.get(CONCEPT.SEQUENCING.MAX_INSERT_SIZE),
                record.get(CONCEPT.SEQUENCING.MEAN_DEPTH),
                record.get(CONCEPT.SEQUENCING.MEAN_INSERT_SIZE),
                record.get(CONCEPT.SEQUENCING.MEAN_READ_LENGTH),
                record.get(CONCEPT.SEQUENCING.TOTAL_READS),
                record.get(CONCEPT.GENOMIC_FILE.HARMONIZED),
                record.get(CONCEPT.GENOMIC_FILE.REFERENCE_GENOME),
            ]
        )
```